sdk/state/integration/helpers_test.go

```diff
@@ -178,7 +216,53 @@ func retry(maxAttempts int, f func() error) (err error) {
 	return err
 }
 
-func fund(client horizonclient.ClientInterface, account *keypair.FromAddress, startingBalance int64) error {
+func fundAsset(client horizonclient.ClientInterface, asset txnbuild.Asset, amount int64, accountKP *keypair.Full, distributorKP *keypair.Full) error {
```

Comment:
a new method. This method will fund an account with a specific asset. This is needed so that we can fund an account with a non-native asset to test with. Before the funding came from the `fund` method, which only created an account with a starting xlm balance